codecov.yml

```diff
@@ -0,0 +1,20 @@
+codecov:
+  branch: 
+
+coverage:
+  status:
+    project:
+      default:
+        # minimum coverage ratio that the commit must meet to be considered a success
+        target: 45%
+        if_ci_failed: error
+        only_pulls: true
+    patch:
+      default:
+        # minimum coverage ratio that the commit must meet to be considered a success
+        target: 45%
```

Comment:
Should we use a target so low for patches? I think it's better to have a higher target for patches to stimulate all new PRs to be merged only with enough coverage.

Reply:
@luislhl 

I agree, increased the coverage to 90% on 636fbd2639ee7192c4190ce76e3bbee106635877

Reply:
I agree, refactored it to 80% on cea0ff8c546b1e7afcb1c54f685bbed88fb29952